src/js/edu-benefits/5490/config/form.js

```diff
@@ -406,8 +405,9 @@ const formConfig = {
                 'ui:options': {
                   hideIf: form => {
                     const status = _.get('highSchool.status', form);
-                    return status !== 'graduationExpected';
-                  }
+                    return status !== 'discontinued';
```

Comment:
From the latest updates to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1793
it sounds like the only selection for which the date is not needed is `neverAttended`

And that the additional high school data should be collected if the selection is either `graduated ` _or_ `discontinued`, but in the latter case, only `name`, `city`, and `state` are needed, whereas if they graduated we need everything, `hours, hoursType, degreeReceived`. (And that `dateRange` is never needed, so fine to remove completely.)

(Aside: it seems kind of weird to ask for `hours` for high school, but I guess if the user chooses Semester, it makes more sense.)